eta/core/web.py

```diff
@@ -150,12 +153,16 @@ def write(self, path, url, params=None):
             self._do_download(r, f)
 
     def _get_streaming_response(self, url, headers=None, params=None):
+        #get the defualt system proxy settings by default
+        proxies = urllib.request.getproxies()
```

Comment:
Would it make more sense to register the proxies only once when `sess` is created in the class constructor?